interface/.dockerignore

```diff
@@ -0,0 +1,4 @@
+bin
+build
+node_modules
```

Comment:
Are these `node_modules` something that were created by `npm` and now can be cleaned up since `npm` isn't installing packages on the host machine anymore? (I don't see the file/directory within the repo.)